### DISCUSSION TITLE
XML Resource Routes

### DISCUSSION THREAD

**Discussion by piotrkulpinski** [sergiodxa/remix-utils#135] [Ideas]
Hi, 

Any chance to add a XML Helper function to create a XML file responses.
That would be helpful for creating sitemaps and similar content easier. 

Thanks!

**Comment by sergiodxa** [comment]
Do you want to send a PR? It should be similar to other response helpers, just ensure you set the correct Content-Type

**Comment by piotrkulpinski** [comment]
PR created: https://github.com/sergiodxa/remix-utils/pull/150

Would you mind if I add another one for TXT files? Both would be helpful for creating robots.txt and sitemap.xml files.

**Reply by sergiodxa** [reply]
Yes that would be useful, I think `text/plain` is the content-type?

**Reply by piotrkulpinski** [reply]
That is correct. Just pushed another PR. Would be nice to abstract those response functions into a helper function as most of it is just repeated.
